test/integration/http/_utils.js

```diff
@@ -164,6 +165,52 @@ function rmPublic (t) {
   }
 }
 
+let activeSideChannel
+let makeSideChannel = async (port = 3433) => {
```

Comment:
Totally bikesheddy / stylistic but since I think `this` isn't really a factor here, I tend to prefer named function expressions

Reply:
Ya, that's what I'm saying!